examples/nodejs/lambda-examples/cloudwatch-metrics/README.md

```diff
@@ -0,0 +1,119 @@
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+<br>
+
+## CloudWatch Metrics using Momento's Experimental Metrics Middleware
+
+This directory contains an AWS CDK stack that will create and populate a custom CloudWatch dashboard for Momento client-side metrics. You can deploy an example Lambda function or ECS cluster that uses the Momento Node.js SDK's Experimental Metrics Middleware to generate example dashboard data for 5 minutes at a time. Or you can deploy just the CloudWatch dashboard and use your own application to emit Momento metrics.
+
+## Prerequisites
+
+- Node version 18 or higher is required
+- To get started with Momento, you will need a Super User Momento API key. You can get one from the [Momento Console](https://console.gomomento.com). Check out the [getting started](https://docs.momentohq.com/getting-started) guide for more information on obtaining an API key.
+- To deploy the CDK app, you will need to have [configured your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-authentication.html#cli-chap-authentication-precedence).
+
+## Deploying the Lambda Function
+
+Follow the instructions in this section to deploy a CloudWatch dashboard along with a Node.js Lambda function that uses Momento's Experimental Metrics Middleware.
+
+First make sure to start Docker and install the dependencies in the `lambda` directory, which is where the Lambda code lives.
+
+```bash
+cd lambda
+npm install
+```
+
+The source code for the CDK application lives in the `infrastructure` directory.
+To build and deploy it you will first need to install the dependencies:
+
+```bash
+cd infrastructure
+npm install
+```
+
+Then create a `.env` file in the `infrastructure` directory, which is where you will define all required configuration variables. 
+You will need the Super User API key you generated from the [Momento Console](https://console.gomomento.com). You will also set the `STACK_CONFIG` variable to `lambda` like so:
+
+```bash
+MOMENTO_API_KEY=<YOUR_MOMENTO_API_KEY>
+STACK_CONFIG="lambda"
```

Comment:
nit - can we rename this var to something like `EXAMPLE_MOMENTO_APPLICATION` or `ADDITIONAL_APPLICATION` or something along those lines?